crypto/ml_dsa/ml_dsa.h

```diff
@@ -36,6 +36,10 @@ OPENSSL_EXPORT int ml_dsa_44_keypair(uint8_t *public_key,
 OPENSSL_EXPORT int ml_dsa_44_pack_pk_from_sk(uint8_t *public_key,
                                              const uint8_t *private_key);
 
+OPENSSL_EXPORT int ml_dsa_44_keypair_internal_no_self_test(uint8_t *public_key,
```

Comment:
Do these new functions need to be exported? I don't see anything outside of libcrypto calling them.

Reply:
removed